modules/importing-repositories/procedure-integrating-bulk-import-with-orchestrator-workflows.adoc

```diff
@@ -0,0 +1,66 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="integrating-bulk-import-with-orchestrator-workflows"]
+
+= Setting up a custom scaffolder workflow for bulk import
+
+As an administrator, you can create a scaffolder template specifically for the Bulk Import plugin. You can provide the scaffolder template within the configuration of your application.
+
+You can perform the following tasks with this integration:
+
+* Import existing catalog entities
+* Create pull requests for cleanup
+* Call webhooks for external system integration
+
+The bulk import plugin allows you to execute a custom automation workflow of a Scaffolder template on multiple selected repositories.
+
+.Prerequisites
+
+* Create a Scaffolder template specifically designed for the Bulk Import function and provide the scaffolder template within the configuration of your application. This template is the core definition of your customized import scenario.
+
+.Procedure
+
+* Configure the following fields in your {my-app-config-file} configuration to instruct the Bulk Import plugin to use your custom templates:
+
+** `importTemplate`
+** `importAPI`
+** `open-pull-requests`
+** `scaffolder`
+
+you can direct the Bulk Import plugin to hand off the entire list of selected repositories to a custom Orchestrator workflow.
+To configure the Bulk Import workflow:
++
+[source,yaml]
+----
+bulkImport:
+  importTemplate: your-template-entity-reference-or-template-name
+  importAPI: 'scaffolder', 'orchestrator'
```

Comment:
Consistency: `open-pull-requests` and  `scaffolder` mentionned above and below are missing in the example. Consider using an example with all mentionned fields.